server/odc-server/src/main/java/com/oceanbase/odc/server/web/controller/v2/DatabaseChangeController.java

```diff
@@ -47,46 +54,70 @@
 @RequestMapping("/api/v2/databasechange")
 public class DatabaseChangeController {
     @Autowired
-    private DatabaseChangeChangingOrderTemplateService databaseChangeChangingOrderTemplateService;
+    private DatabaseChangeChangingOrderTemplateService templateService;
 
     @ApiOperation(value = "createDatabaseChangingOrderTemplate", notes = "create database changing order template")
     @PostMapping("/changingorder/templates")
     public SuccessResponse<Boolean> createDatabaseChangingOrderTemplate(
```

Comment:
create method return the created object
update method return the updated object
delete method return the deleted object

Reply:
have done